mapdata/WarcraftLegacies/UpgradeData/Skin/925910098.json

```
@@ -0,0 +1,53 @@
+{
+  "OldId": 1802331474,
+  "NewId": 925910098,
+  "Unk": [
+    0
+  ],
+  "Modifications": [
+    {
+      "Level": 1,
+      "Id": 1835101799,
+      "Type": 3,
+      "Value": "Improved Crystal Discharge"
+    },
+    {
+      "Level": 1,
+      "Id": 829579623,
+      "Type": 3,
+      "Value": "ReplaceableTextures\\CommandButtons\\BTNEarthCrystalElement.blp"
+    },
+    {
+      "Level": 1,
+      "Id": 828536167,
+      "Type": 3,
+      "Value": "Improve the stun duration for Crystal Discharge from 0.15 and 1 to 0.30 and 2"
```

That is crazy confusing to read!
```suggestion
      "Value": "Improves Crystal Discharge by increasing its stun duration against units to 1 and its stun duration against heroes to 2."
```